src/common/index.ts

```diff
@@ -1,19 +0,0 @@
-import contractNetworks from '@cowprotocol/contracts/networks.json'
-import { ALL_SUPPORTED_CHAIN_IDS } from './chains'
-
-const { GPv2Settlement } = JSON.parse(contractNetworks as unknown as string) as typeof contractNetworks
-
-export const COW_PROTOCOL_SETTLEMENT_CONTRACT_ADDRESS = ALL_SUPPORTED_CHAIN_IDS.reduce<Record<number, string>>(
```

Comment:
Is this not used externally somewhere else? Maybe mark as deprecated?

Reply:
I haven't found it used in CowSwap and Explorer both.
Maybe somebody has used it, however, they can use it from '@cowprotocol/contracts'.
I would like to discuss relations between SDK and @cowprotocol/contracts, thanks for bringing up the topic!